mobile/src/features/Notifications/common/hooks.ts

```diff
@@ -108,19 +107,68 @@ const initPushNotifications = (
     },
   )
 
-  const responseListener =
-    Notifications.addNotificationResponseReceivedListener((_response) => {
-      const id = parseNotificationId(Date.now().toString())
-      triggerNotificationAction({
+  const processOpenData = async (
+    data: Record<string, unknown> | undefined,
+    title?: string,
+    body?: string,
+  ) => {
+    if (!data) return
+    if (typeof data === 'object' && data !== null) {
+      logger.info('Notification opened with data', {data, title, body})
+      const id = Date.now()
+      const pushEvent = createPushNotification({
```

Comment:
This is push as well, look into the push notifications PR to make sure it's necessary